dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -962,12 +962,12 @@ def create_admin_script
     assert_response :forbidden
   end
 
-  test "should get show of admin script if signed in as admin" do
+  test "should not get show of admin script if signed in as admin" do
```

Comment:
No one. :) The `admin_only` and `student_of_admin` scripts are being deprecated and removed (as part of this work). Adding a TODO to remove (consolidate) this (these) test(s), though for now, I'm keeping the test as it is "useful".